protocol-definitions/Experimental.yaml

```diff
@@ -1,3 +1,31 @@
 id: 253
 name: Experimental
-methods: []
\ No newline at end of file
+methods:
+  - id: 1
+    name: pipelineSubmit
```

Comment:
The existing JET message is `submitJob` and maybe we want that one to follow a similar name such as `submitYamlJob`? But then.... that existing message is different since it accepts the `jobId` as a parameter. And no binary data.

Also it accepts the `lightJobCoordinator` which we may want to enable here too?

Reply:
I am OK with naming if nobody rejects it, can you comment in the TDD about this naming suggestion?

Reply:
Existing message is different since it is a different message. We return the job Id in the response once the job is started.

Why do you think the `lightJobCoordinator` is needed for our case?

Reply:
Because you may want to submit a job using YAML and set a `lightJobCoordinator` just the way you would do with Java?

Reply:
Please see limitations of a light job at [here](https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/jet/JetService.java#L166). Do we really want to support it at phase 1 from today? If a request comes later, surely we can add. The `lightJobCoordinator` UUID requires that the user uses a specific UUID for a specific member but at Viridian, we manage the cluster and the members, the user normally should not deal with individual members, right? 

do you still think we need this parameter?